supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesAsyncBackendJobExecutionActor.scala

```diff
@@ -80,11 +98,20 @@ class JesAsyncBackendJobExecutionActor(override val standardParams: StandardAsyn
 
   private lazy val dockerConfiguration = jesConfiguration.dockerCredentials
 
+  private val previousRetryReasons: ErrorOr[PreviousRetryReasons] = PreviousRetryReasons.tryApply(jobDescriptor.prefetchedKvStoreEntries, jobDescriptor.key.attempt)
+
+  override val preemptible: Boolean = previousRetryReasons match {
+    case Valid(PreviousRetryReasons(p, ur)) => p < maxPreemption
+    case _ => false
+  }
+
   override def tryAbort(job: StandardAsyncJob): Unit = {
     Run(job, initializationData.genomics).abort()
   }
 
-  override def receive: Receive = pollingActorClientReceive orElse runCreationClientReceive orElse ioReceive orElse super.receive
+  override def requestsAbortAndDiesImmediately: Boolean = true
+
+  override def receive: Receive = pollingActorClientReceive orElse runCreationClientReceive orElse ioReceive orElse kvClientReceive orElse super.receive
```

Comment:
Yeah this is something we're going to need to figure out at some point as it's pretty much impossible to track stuff down now, but it is what it is at the moment